data-node/src/main/java/org/graylog/datanode/management/OpensearchProcessImpl.java

```diff
@@ -55,16 +56,18 @@ class OpensearchProcessImpl implements OpensearchProcess, ProcessListener {
 
     private final Queue<String> stdout;
     private final Queue<String> stderr;
+    private final CaService caService;
 
-    OpensearchProcessImpl(DatanodeConfiguration datanodeConfiguration, int logsCacheSize) {
+    OpensearchProcessImpl(DatanodeConfiguration datanodeConfiguration, int logsCacheSize, final CaService caService) {
```

Comment:
I believe it is problematic that this class is not unit-tested at all...

Reply:
I wrote an issue for that so that we address it later